frontend/src/search/Search.css

```diff
@@ -0,0 +1,55 @@
+.Search {
+    position: fixed;
+    top: 30px;
+    left: 30px;
+    width: 300px;
+    background-color: white;
+    border-radius: 5px;
+    box-shadow: 0px 0px 17px 0px rgba(0,0,0,0.2);
+    padding: 10px;
+}
+
+.From, .To {
```

Comment:
Is it possible to use SCSS perhaps? Maybe for later in the project

Reply:
Great suggestion; I've added it to the backlog!